data/sql/updates/pending_db_world/newmans-landing.sql

```diff
@@ -0,0 +1,15 @@
+DELETE FROM `npc_vendor` WHERE `entry` = 26081;
+INSERT INTO `npc_vendor` (`entry`, `slot`, `item`, `maxcount`, `incrtime`, `ExtendedCost`, `VerifiedBuild`) VALUES
+(26081, 2, 2880, 0, 0, 0, 0),
+(26081, 0, 2901, 0, 0, 0, 0),
+(26081, 3, 3466, 0, 0, 0, 0),
+(26081, 5, 3857, 0, 0, 0, 0),
+(26081, 1, 5956, 0, 0, 0, 0),
+(26081, 4, 18567, 0, 0, 0, 0);
+
+UPDATE `creature_equip_template` SET `ItemID1` = '19022' WHERE  `CreatureID` = 26081;
+UPDATE `creature_equip_template` SET `VerifiedBuild` = '0' WHERE  `CreatureID` = 26081;
```

Comment:
Don't use single quotes around integers, please :)

Reply:
Got rid of the quotes, sorry about that :D 
Edited verified build because I figured that was the correct thing to do, seeing as the information being verified is being edited. I would have set it to the build I sniffed, but that number was too high for the datatype.